mapswipe_workers/tests/unittests/set_up.py

```diff
@@ -0,0 +1,76 @@
+"""Helper functions for test set up"""
+
+import json
+import os
+
+from mapswipe_workers import auth
+
+
+def set_firebase_test_data(
+    data_type: str, identifier: str, project_type: str = "",
```

Comment:
what do `data_type` and `identifier` refer to?

Reply:
data_type is either users, projects, tasks, groups or results.
Identifier is either user_id, project_id, etc.